package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@buildit/gravity-ui-sass",
-  "version": "0.11.1",
```

Comment:
I saw that in the FAQ. But I can't seem to find an option that allows the git plugin to overwrite the master and develop branch. Maybe they removed that functionality or I can't find it 🤔

Reply:
Further research I might have been wrong. After re-reading the docs several times 🤦‍♂️ (it's quite vague) looks like I can include the `git` plugin manually and it should do this automatically. I thought it was the same as the GitHub plugin, but they're different.

Reply:
Going to run some tests on my other branch, but beyond that looks good to role. Will resolve this convo if everything looks good in the test repo.

Reply:
Here is what the GIT app will do, it creates a new push to `master` only. It includes the version, changelog, and some other good things to check into `master`. I don't see a way to make it also push to `develop` (which I think is fine).

![screen shot 2019-02-27 at 1 49 03 pm](https://user-images.githubusercontent.com/643307/53522608-fc5df700-3a97-11e9-9c26-682c98918d1f.png)

Reply:
PS On GitHub it looks like we'll have to whitelist the corresponding branch to allow it to push. As it looks like there are safeguards requiring an approved PR to push (including admins). I'm guessing that it should be fine if Travis already has something built in to circumvent the permission.

Reply:
Sweet.

My preference then would be to use that git plugin so that those useful bits get pushed back to our repo. I'm not fussed about it not going back to `develop` for now. For now we can do so manually (I don't anticipate us releasing so frequently that it would become a big burden). Perhaps a future PR can look into automating that somehow.

As for the permissions - feel free to adjust the repo's settings as needed. Perhaps we can allow only the `buildit-dev` account (or whichever one Travis has the credentials for) to push to master. I think some restrictions are better than none simply to guard against human error.

Reply:
This might actually already work depending on how the `buildit-dev` token was configured. I'm going to tinker with my fork branch today to see what we need to do (no clear instructions on this subject anywhere). Will update here with the results.

Reply:
After doing a little research it looks like the only way to really do this is to remove the "required by administrator" hook. We may need to adjust how users are setup to minimize potential problems. My recommendation is as follows:

* Create an admin group limited to primary admins and for bots such as buildit-dev (with `admin` access)
* Move everyone else to a group with `write` access
* Remove enforce admin on `master` branch

This seems to be the intended design pattern Git wants people to use. Also not enforcing admin still requires PRs to be forcefully merged. So there are still safeguards in place.

Reply:
That sounds good to me.
Presumably this is at the level of "teams" within the Buildit org, right? So, instead of just https://github.com/orgs/buildit/teams/gravity-maintainers, we'd have 2 teams as you propose.

If so, I'm fine with that. Feel free to set them up. I think for the admins one, the bot account, you and me would suffice for now.

Reply:
Yep, exactly what I'm thinking. I'll get that setup so we can merge this in today 👍